src/App.tsx

```diff
@@ -22,23 +25,45 @@ initAnalytics();
 
 function App() {
   globalStyles();
+  const isCoLinks = useIsCoLinksSite();
+
+  // TODO: add this when we have a good favicon
```

Comment:
@wingfeatherwave @CryptoGraffe I want to show notifications count in favicon!!!!